testing/spectest/shared/common/forkchoice/runner.go

```diff
@@ -41,7 +41,7 @@ func init() {
 // Run executes "forkchoice"  and "sync" test.
 func Run(t *testing.T, config string, fork int) {
 	runTest(t, config, fork, "fork_choice")
-	if fork >= version.Bellatrix {
+	if fork >= version.Bellatrix && fork < version.Fulu {
```

Comment:
What's up with this? No sync tests in fulu?

Reply:
No sync test on fulu. Nothing changed there but clarifying that with the spec test team at this moment

Reply:
Any update?

Reply:
This was added in latest spec test. Let me update spec test version in another PR first